Lifecycle.md

```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
 and other important getting-started documents, please see our 
 [examples](Bootstrap-Policies/).
 
-## Submitting an Application
+## Membership Application and Approval Process
 
-The application process to join the Planetary Software Organization
+The Planetary Software Organization is quite new and currently has
+limited resources available to mentor new projects. As such, projects
+are chosen for admission as mentors become available.
+
+You can apply at any time and the TSC and available mentors will
+help improve your application while awaiting available resources.
+
+The process to join the Planetary Software Organization
 as a TLP or TLWG is as follows.
 
+### 1. Application Completed - Pull Request Submitted
```

Comment:
I am finding the 'Application Completed' to be weird here. If the PR is submitted is the application complete? It seems like that just starts the process.?

Reply:
Yeah, maybe "Submit Application" here?

Reply:
Updated. Do we like the wording?